src/sentry/static/sentry/app/views/settings/organizationIntegrations/configureIntegration.tsx

```diff
@@ -135,16 +145,76 @@ class ConfigureIntegration extends AsyncView<Props, State> {
           )
         )}
 
-        {provider && provider.features.includes('alert-rule') && (
+        {provider.features.includes('alert-rule') && (
           <IntegrationAlertRules integration={integration} />
         )}
 
-        {provider && provider.features.includes('commits') && (
+        {provider.features.includes('commits') && (
           <IntegrationRepos {...this.props} integration={integration} />
         )}
       </React.Fragment>
     );
   }
+
+  renderBody() {
+    const {integration} = this.state;
+    const provider = this.state.config.providers.find(
+      p => p.key === integration.provider.key
+    );
+    if (!provider) {
+      return null;
+    }
+
+    const title = <IntegrationItem integration={integration} />;
+    const header = (
+      <SettingsPageHeader noTitleStyles title={title} action={this.getAction(provider)} />
+    );
+
+    return (
+      <React.Fragment>
+        {header}
+        {this.renderMainContent(provider)}
+      </React.Fragment>
+    );
+  }
+
+  //renders everything below header
+  renderMainContent(provider: IntegrationProvider) {
```

Comment:
@scefali so just to be clear, we are changing how all the integration configs are going to look, but the change for non flagged in stack trace link integrations will just be the singular tab view, is that correct?

Reply:
@MeredithAnya correct